src/dropdown/styles.ts

```diff
@@ -107,31 +108,38 @@ export function itemStyle({
   isActive,
   inverted,
 }: ItemStyleOptions) {
-  return css`
-    ${itemBaseStyle}
-
-    min-height: ${sizeMap[size].minHeight};
-    color: ${inverted ? tokens.colors.white : tokens.colors.navy};
-    background: ${isActive
-      ? inverted
-        ? tokens.colors.transparentWhiteSoft
-        : tokens.colors.transparentGreySoft
-      : 'transparent'};
-
-    ${!isActive &&
-    css`
-      &:hover {
-        background-color: ${inverted
-          ? tokens.colors.transparentWhiteSubtle
-          : tokens.colors.transparentGreySubtle};
-      }
-    `}
-
-    ${isFocusVisible &&
-    `box-shadow: inset 0 0 0 2px ${tokens.colors.blueDark};`}
-  `;
+  return [
+    itemStaticStyle,
+    css({
+      minHeight: sizeMap[size].minHeight,
+      color: inverted ? tokens.colors.white : tokens.colors.navy,
+      background: isActive
+        ? inverted
+          ? tokens.colors.transparentWhiteSoft
+          : tokens.colors.transparentGreySoft
+        : 'transparent',
+
+      ...(!isActive && {
+        '&:hover': {
+          backgroundColor: inverted
+            ? tokens.colors.transparentWhiteSubtle
+            : tokens.colors.transparentGreySubtle,
+        },
+      }),
+
+      ...(isFocusVisible && {
+        boxShadow: `inset 0 0 0 2px ${tokens.colors.blueDark}`,
+      }),
+    }),
```

Comment:
The `css` function takes as many arguments as you want, so you could rewrite this without object spreading, which is a bit less indented (I think?) & less syntax-heavy:

```suggestion
    css({
      minHeight: sizeMap[size].minHeight,
      color: inverted ? tokens.colors.white : tokens.colors.navy,
      background: isActive
        ? inverted
          ? tokens.colors.transparentWhiteSoft
          : tokens.colors.transparentGreySoft
        : 'transparent',
    },
    !isActive && {
      '&:hover': {
        backgroundColor: inverted
          ? tokens.colors.transparentWhiteSubtle
          : tokens.colors.transparentGreySubtle,
      },
    },
    isFocusVisible && {
      boxShadow: `inset 0 0 0 2px ${tokens.colors.blueDark}`,
    }),
```